### PR TITLE
Fix `save` testset

### LINKED ISSUE #34: Fix save issues in test suite
The tests in the "save" test set are failing with some combinations of file formats. We need to fix them all.

### PR DESCRIPTION
**Pull request by eliascarv**
closes #34

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -248,24 +251,38 @@ savedir = mktempdir()
...
+      c1 = Tables.columns(t1)
+      c2 = Tables.columns(t2)
+      n1 = Tables.columnnames(c1)
+      n2 = Tables.columnnames(c2)
+      @test Set(n1) == Set(n2)
+      for n in n1
+        x1 = Tables.getcolumn(c1, n)
+        x2 = Tables.getcolumn(c2, n)
+        @test x1 == x2
+      end
```
Comment (juliohm) [inline test/runtests.jl:285]: Cant we just use gt1 == gt2?

File: test/runtests.jl [hunk 2]
```diff
@@ -248,24 +251,38 @@ savedir = mktempdir()
       "land.shp",
       "path.shp",
       "zone.shp",
-      "field.kml",
```
Comment (juliohm) [inline test/runtests.jl:251]: can you elaborate?

File: test/runtests.jl [hunk 3]
```diff
@@ -248,24 +251,38 @@ savedir = mktempdir()
...
       f1 = joinpath(datadir, fname)
-      f2 = joinpath(savedir, first(splitext(fname)) * fmt)
+      f2 = joinpath(savedir, replace(fname, "." => "-") * fmt)
 
       # load and save table
-      # t1 = GeoTables.load(f1)
-      # GeoTables.save(f2, t1)
-      # t2 = GeoTables.load(f2)
-      # @test t1 == t2
+      kwargs = endswith(f1, ".geojson") ? (; numbertype=Float64) : ()
```
Comment (juliohm) [inline test/runtests.jl:264]: Is this a limitation of geojson format?

**Review by juliohm — CHANGES_REQUESTED** [review]

File: test/runtests.jl (hunk 3 above)
Comment (eliascarv) [reply test/runtests.jl:264]: Yes

**Review by juliohm — APPROVED** [review]

**Pull request merged**